app/models/user.rb

```diff
@@ -46,6 +46,8 @@ class User < ApplicationRecord
   validates :password, length: { within: 10..200 }, allow_nil: true, unless: :skip_password_validation?
   validate :unconfirmed_email_uniqueness
 
+  enum mfa_level: { no_auth: 0, auth_only: 1, auth_and_write: 2 }
```

Comment:
Would renaming `no_auth` to `mfa_disabled` or just `disabled` make it easier to understand?

Reply:
Since Rails enums generates methods like `no_auth?` and `no_auth!` for class `User` automatically, I think using `disabled` will be a little bit ambiguous

Reply:
`no_auth` is not representing (mfa being disabled) what is says it is (no_auth ~ no authentication?).

Reply:
(Also reply for the comment on mfa/two_factor_auth names)
Are `tfa` and `two_factor_auth` easily recognized as the same? If so, I think something like `no_tfa` is better name.

Reply:
Because `two_factor_auth` and `two_factor_authentication` is too lengty...

Reply:
`no_tfa` or `no_mfa` is better.